feature/selector.py

```diff
@@ -24,6 +24,8 @@
 from sklearn.ensemble import GradientBoostingClassifier, GradientBoostingRegressor
 from sklearn.model_selection import KFold
 from xgboost import XGBClassifier, XGBRegressor
+from joblib import Parallel, delayed
+import multiprocessing as mp
```

Comment:
Move `multiprocessing` import to section at top with the built-in libraries (`time`, `typing`, etc.)